src/components/Snackbar/SnackbarDismissAction.test.tsx

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2024 Nordeck IT + Consulting GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { act, render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { ComponentType, PropsWithChildren } from 'react';
+import { SnackbarDismissAction } from './SnackbarDismissAction';
+import { SnackbarContextValue, SnackbarProvider } from './SnackbarProvider';
+import { useSnackbar } from './useSnackbar';
+
+describe('<SnackbarDismissAction />', () => {
+  let snackbarContextValue: SnackbarContextValue;
+  let SnackbarProviderTest: ComponentType<PropsWithChildren<{}>>;
+
+  beforeEach(() => {
+    function SnackbarContextValueExtractor() {
+      snackbarContextValue = useSnackbar();
+      return null;
+    }
+
+    SnackbarProviderTest = () => {
+      return (
+        <SnackbarProvider>
+          <SnackbarContextValueExtractor />
+        </SnackbarProvider>
+      );
+    };
+  });
```

Comment:
Could we use a wrapper way we do in other tests?

Reply:
In this case we can't because the test interacts with the frontend. That is not available if we use `renderHook`. :shrug: (Also possible that I am doing something wrong here).

Reply:
Currently the Snackbar is added by SnackbarProvider and that is why it is a bit implicit in the tests. If we could have a separate component for Snackbar and then explicitly add it to the app and to the tests in render function with wrapper would make it to like more how it is done in other widgets.

Reply:
I've extracted the snackbar rendering